packages/api-reference/src/components/Content/Lazy/Loading.vue

```diff
@@ -218,7 +218,7 @@ onMounted(() => {
               <Anchor :id="getModelId({ name })">
                 <SectionHeaderTag :level="2">
                   {{
-                    (getModels(parsedSpec)?.[name] as OpenAPIV3.SchemaObject)
+                    (getModels(parsedSpec)?.[name] as OpenAPIV3_1.SchemaObject)
```

Comment:
one step up would be type it in getModels, but also we will remove this component anyway